lib/config.js

```diff
@@ -53,19 +58,13 @@ config.mongodb.connectOptions = {
   promoteBuffers: true,
 };
 config.mongodb.writeOptions = {
-  j: true,
-  w: 'majority',
-  // as mongo 4.2 this is no longer used by updateOne or updateMany
-  // FIXME remove this once all dependencies have been upgraded
-  multi: true,
+  writeConcern,
   // this is used by insert methods
   forceServerObjectId: true,
 };
 /* DEPRECATED */
 config.mongodb.options = {
```

Comment:
has been deprecated since version `2.0.0` so probably safe to remove.

Reply:
But do we even use it anywhere? If not, it should definitely be removed. If we're using it, then it will take another major rev if we remove it now -- but we probably should so we can stop maintaining it.

Reply:
I am unable to find anywhere in our code base it is used. There could be a developer external to use using it. This is a major oversite though as we are on version 8, and it has been marked deprecated since version 2.

Reply:
The options are limited to our usage in this module, so if this module doesn't use them, they can be safely removed. Let's remove them now.

Reply:
Removed here: https://github.com/digitalbazaar/bedrock-mongodb/pull/65/commits/7eeb37cb9491e087f06579ea8ecd1fe80cbfedae